modal.css

```diff
@@ -0,0 +1,251 @@
+.modal-popup {
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
+  background-color: #191765;
+  background-image: url('./image/Background-image/top-circles.png');
+  background-repeat: no-repeat;
+  background-size: contain;
+  box-sizing: border-box;
+  opacity: 0;
+  visibility: hidden;
+  transform: scale(1.1);
+  transition: visibility 0s linear 0.25s, opacity 0.25s 0s, transform 0.25s;
+  display: flex;
+  justify-content: center;
+}
```

Comment:
- [ ] Kindly make sure your popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g.,: **ON DESKTOP:** spacing around and between elements in the popup, height, spacing around the popup itself(bottom, top,...), etc), **ON Mobile:** some elements are not visible(buttons at the bottom),... Please check all design details, and make sure you stick your popup to the Figma template for DESKTOP && Mobile. I suggest you add the **overflow-y: auto** in the `.modal-popup { ... }` class and hide the **the scrolling** of the sub-background content so the user can not interact with the sections when he is watching the popup, this will alloy you to easily update your styling.
####  Figma design:
![Screenshot from 2022-06-24 01-42-28](https://user-images.githubusercontent.com/83042742/175434022-6b4f7841-acd9-45a1-a033-70409eafdf10.png)
####  Your design:
![Screenshot from 2022-06-24 01-58-57](https://user-images.githubusercontent.com/83042742/175434409-32f705b9-b223-428e-bf46-86ab49ddf1ad.png)